default/stringtables/en.txt

```diff
@@ -11440,6 +11462,9 @@ AVERAGE_POPULATION
 GOOD_POPULATION
 +	Good Population 125%
 
+FIXED_LOW_POPULATION
+−−	Population is fixed at 5
```

Comment:
population or target population?

if the actual population should also be fixed, you need an extra effect to set it as such...